docs/antrea-network-policy.md

```diff
@@ -108,16 +111,45 @@ still enforced before K8s NetworkPolicies. Thus, admin-created tiered Antrea
 Policy CRDs have a higher precedence than developer-created K8s
 NetworkPolicies.
 
+### kubectl commands for Tier
+
+The following kubectl commands can be used to retrieve Tier CRDs:
```

Comment:
as you pointed out to me earlier, the term CRDs is not totally accurate here. I think we should use "Tier resources" maybe?

Reply:
done